tests/unit/test_settings_blueprint.py

```
@@ -0,0 +1,1076 @@
+        finally:
+            mod._set_update_state(False, None)
+
+    def test_start_update_systemd_path(self, client, monkeypatch):
+        """POST /settings/update uses systemd path when available."""
+        import blueprints.settings as mod
+
+        monkeypatch.setattr(mod, "_systemd_available", lambda: True)
+        monkeypatch.setattr(
+            mod, "_get_update_script_path", lambda: "/fake/update.sh"
+        )
+        monkeypatch.setattr(mod, "_start_update_via_systemd", lambda u, s: None)
```

Ruff (0.15.7)

[warning] Unused lambda argument: `u`

(ARG005)